proposals/rcp-035-single-feeds.md

```diff
@@ -0,0 +1,141 @@
+# RESO (Endorsement Name) Endorsement
+
+| **RCP** | 035 |
+| :--- | :--- |
+| **Version** | **DD 2.1** |
+| **Authors** | Eric Finlay |
+| **Specification** | [RCP-035](https://github.com/RESOStandards/transport/issues/96) |
+| **Status** | IN PROGRESS |
+| **Date Ratified** | Pending |
+| **Dependencies** | None |
```

Comment:
This will be whatever version of DD it's added to, I think it can be DD 2.0 since the close date is Oct 31 and this proposal is well-vetted. What we don't have yet are "complex" types for the `FeedRules` part mentioned in the original post. 

That said, we could still put it in the spec and make it clear that we wouldn't test the `FeedRules` part yet.

Here it is from the discussion: 

```
{
  "StandardStatus": "Pending",
  "PrivateRemarks": "This is an example remark",
  "FeedTypes": ["IDX", "BBO"],
  "FeedRules": [
    {
      "FieldName": "PrivateRemarks",
      "FeedType": "IDX",
      "Visible": true
    }
  ]
}
```

Reply:
I've added `FeedRules`. I left the target spec as DD 2.1 since it's now past Oct 31. I figure we can hash that out with DD.